install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -40,19 +40,26 @@ spec:
       for: 10m
       labels:
         severity: critical
+    - alert: CannotRetrieveUpdates
+      annotations:
+        message: Cluster version operator has not retrieved updates since {{ "{{ $labels.lastTime }}" }}. Failure reason {{ "{{ $labels.reason }}" }}. {{ "{{ $labels.message }}" }}.
+      expr: |
+        cannot_retrieve_updates{job="cluster-version-operator"} == 1
```

Comment:
We don't really need `cannot_retrieve_updates` metric for that as it can be derived from `cluster_version_operator_successful_update_retrieval`:

```
time() - cluster_version_operator_successful_update_retrieval > 3600
```